app/controllers/state_file/archived_intakes/email_address_controller.rb

```diff
@@ -0,0 +1,38 @@
+module StateFile
+  module ArchivedIntakes
+    class EmailAddressController < ArchivedIntakeController
+      before_action :check_feature_flag
+      def edit
+        @form = EmailAddressForm.new
+      end
+
+      def update
+        @form = EmailAddressForm.new(email_address_form_params)
+
+        if @form.save
+          archived_intake = StateFileArchivedIntake.find_by(email_address: @form.email_address)
+          session[:email_address] = @form.email_address
+          StateFileArchivedIntakeRequest.find_or_create_by(email_address: @form.email_address, ip_address: ip_for_irs, state_file_archived_intakes_id: archived_intake&.id )
+          create_state_file_access_log(0)
```

Comment:
the nice thing about rails enums is you don't have to refer to them by their integer values, you can use the english word!